### PR TITLE
Improve greeting capabilities

### PR DESCRIPTION
**Pull request by ravench**
Lat's face it: ```Yes?``` gets boring fast.
And dynamic EasterEggs are more fun ;)

### CODE REVIEW

File: master/buildbot/test/unit/test_status_words.py [hunk 1]
```diff
@@ -295,6 +295,13 @@ def test_command_dance(self):
         self.assertTrue(self.sent)  # doesn't matter what it sent
 
     @defer.inlineCallbacks
+    def test_command_hello(self):
+        yield self.do_test_command('hello', exp_usage=False)
+        self.assertEqual(self.sent, ['yes?'])
+        yield self.do_test_command('hello', exp_usage=False)
+        self.assertTrue(self.sent)
```
Comment (djmitche) [inline master/buildbot/test/unit/test_status_words.py]: This will be true even if it doesn't send anything new (since it already sent 'yes?').

**Comment by djmitche** [comment]
This looks fine to me, except for a small fix in the tests.  I don't think this necessarily warrants a release note, but if you'd like to add one please do.

**Comment by ravench** [comment]
This should do it.
I wouldn't mind adding to the relnotes. I'm just not sure how to, since the index.rst in this branch is fairly old already.. (?)

**Comment by sa2ajj** [comment]
Please rebase against master

**Comment by ravench** [comment]
Oh, damn.. Know what, I'll refork. it's just 20 lines of code.

**Pull request closed without merging**